.github/workflows/deploy.yml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches: [ master ]
   pull_request:
-    branches: [ master ]
+    branches: [ master, gulp-implantation ]
```

Comment:
is this change still actual?

Reply:
Yes, it's still actual because of confusing in pull request merge. I've checked it.

Reply:
I doubt it, since you don't use "gulp-implantation" branch anymore, maybe you should replace it with current branch?

Reply:
I've already tried it. It depends on pull request.
![image](https://github.com/IlyaDevN/Horse-club/assets/106959524/fa2b39f2-4324-4f8e-a7f9-750a8f7cb64f)
It deploys the project after changes in PR founded here. I broke the logic after you asking me to make a wrong merge. Now this PR doesn't correspond to reality